doc/can-define-object.md

```diff
@@ -0,0 +1,479 @@
+@module {function} can-define-object
+@parent can-observables
+@collection can-ecosystem
+@group can-define-object/object.behaviors 0 behaviors
+@group can-define-object/object.static 1 static
+@group can-define-object/object.types 2 types
+@group can-define-object/object.other 3 other
```

Comment:
Is something going to go in this? It just shows an empty header in the sidebar now.